src/usr/local/www/services_unbound_advanced.php

```diff
@@ -350,6 +360,14 @@
 			'Allowed networks must be manually configured on the Access Lists tab if the auto-added entries are disabled.');
 
 $section->addInput(new Form_Checkbox(
+	'disable_auto_added_host_entries',
+	'Disable Auto-added Host Entries',
+	'Disable the automatically-added host entries',
+	$pconfig['disable_auto_added_host_entries']
+))->setHelp('By default, the primary IPv4 and IPv6 addresses of the pfsense machine are added as records for the pfSense system domain (System | General Setup | Domain). ' .
```

Comment:
Change "of the pfsense machine" to "of this firewall"
Change "for the pfSense system domain" to "for the system domain of this firewall"
Remove " (System | General Setup | Domain)" or change to "(System > General Setup)"